From c2dcaf31fe24cd052fcc3355ed413a729ee196d3 Mon Sep 17 00:00:00 2001
From: Joey <joey@lineageos.org>
Date: Tue, 21 Aug 2018 15:06:23 +0200
Subject: [PATCH 20/20] lineage: enable roundIcons by default

We're using circular shape for adaptive icons,
by turning roundIcons on we either get
better circular icons (for apps such as Chrome) or
the same icons as before.

Trebuchet will still allow to swap icons shape

Change-Id: If64ecb427b98e099c55fd3eebdb49dad9cb3dbf2
Signed-off-by: Joey <joey@lineageos.org>
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index ec5f4932..ff5a5993 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -87,4 +87,6 @@
     -->
     <integer translatable="false" name="config_wifi_wakeup_available">1</integer>
 
+    <!-- Flag indicating whether round icons should be parsed from the application manifest. -->
+    <bool name="config_useRoundIcon">true</bool>
 </resources>
-- 
2.17.1

